2017 guidelines

lonsbio opened the pull request:
Have added some recent requirements (e.g ORCID) as outlined on UoM [Graduate Research Hub](http://gradresearch.unimelb.edu.au/preparing-my-thesis/writing-the-thesis#formats-limits) guidelines.

lonsbio committed:
Replace PARstart with IEEEPARstart

lonsbio committed:
Add ORCID

lonsbio committed:
Update front page and front matter to match rules

lonsbio committed:
Remove second reference to PhD, change spacing on title page

lonsbio committed:
Add updated PDF

lonsbio committed:
Rephrase fulfillment based on recent examples

lonsbio committed:
Update readme

lonsbio committed:
Update readme spelling

jpap commented on src/style/uomthesis.sty:
```diff
@@ -198,7 +201,9 @@
-					{\@author} \par
+					{\@author} \par 
+						\vspace{1cm}
+					ORCID: {\@orcid} \par
```
Would this be better as `orcid.org/{\@orcid}`?

That way a reader can go directly to the URL and view the metadata for the given ORCID, and using a lower-case URL softens the otherwise all-caps shouting of the acronym on the title page.

Going further, it might look better when reducing the `\vspace` between the name and the ORCID, and placing the ORCID in a slightly smaller font at 80% greyscale.

What do you think?

jpap commented on src/manuscript.tex:
```diff
@@ -41,11 +41,12 @@
 		% Preface
 		\input{preface}
+		
+        % Acknowledgements
```
There's a bunch of spaces vs. tabs issues like this across many of the commits.  Can you please address them? :-)

jpap commented on src/manuscript.tex:
```diff
@@ -61,6 +62,9 @@
 
+        % Third party copyright
+        % placeholder: TODO
+
```
Perhaps drop this until you've got a good example to contribute.

jpap commented on src/style/uomthesis.sty:
```diff
@@ -284,6 +284,31 @@
+}
+
+
 
```
This command looks unchanged; no need to move?

jpap commented on src/style/uomthesis.sty:
```diff
@@ -213,7 +213,7 @@
 					\vspace{2cm}
 
-					Submitted in total fulfilment of the requirements of the degree\par
+					Submitted in Total Fulfillment of the Requirements of the Degree of Doctor of Philosophy \par
```
(Looks like the new sample PDF is missing this change.)

The requirements don't specify title case here; it looks a bit loud in final print.  I'd recommend sticking to sentence-case.

jpap commented on src/style/uomthesis.sty:
```diff
@@ -205,18 +205,18 @@
-					\bigskip
-					{\LARGE Doctor of Philosophy} \par
-				}
+                    Doctor of Philosophy \par
```
It looks better to on the final print having "Doctor of Philosophy" in `\LARGE` text.

jpap commented on src/style/uomthesis.sty:
```diff
@@ -205,18 +205,18 @@
-				}
+                    Doctor of Philosophy \par
 					\vspace{2cm}
-				%\includegraphics[height=3cm]{\@univlogofile} \par
```
No more logo?  Was this part of the requirements change?

Also, without the logo, the vertical spacing/layout looks a little strange.  I'd suggest either dropping the above `\vspace{2cm}` (with the `\LARGE` text change) or adjusting the whitespace so it looks more consistent across the entire page.

jpap reviewed (changes requested):
Thanks for the PR and updates!  Some comments given below for your consideration.  Looking forward to further updates so I can merge.

Separately, are you able to squash your commits to just one?

lonsbio commented:
Have resubmitted as new PR addressing most comments. ORCID is tricky -  I like the idea of a link, but tried with orcid.org/0000-0000 etc tends to dominate the page.
